dask_jobqueue/core.py

```diff
@@ -216,6 +221,10 @@ def __init__(self,
         if memory is None:
             raise ValueError("You must specify how much memory to use per job like ``memory='24 GB'``")
 
+        if shebang is None:
+            raise ValueError("You must specify a batch script interpreter to use"
+                             " like ``#!/usr/bin/env bash``")
```

Comment:
Is this really a problem if there is no shebang? 
What concerns me here is that this exception might break existing jobqueue configurations, where no shebang is declared. From my point of view it shouldn't.

Reply:
We could maybe add a deprecation warning? and then a warning to say that if shebang is None everywhere, automatically set it to `#!/usr/bin/env bash`  to preserve the old configuration

You are right that this breaks existing configs in its current state.  But you cannot submit a batch script to most schedulers without a shebang:
```
sbatch: error: This does not look like a batch script.  The first
sbatch: error: line must start with #! followed by the path to an interpreter.
sbatch: error: For instance: #!/bin/sh
```

Reply:
Wait, I may be wrong about this. I need to try this PR to see if it breaks my config. Maybe it doesn't, default value should be taken from the module default jobqueue.yaml file.

Reply:
Sorry did not find time today, and this will not be possible until friday. You could verify it though, just create a jobqueue.yaml config file, and do not add a shebang line. 
I believe the only way to reach that code is to create a local config file, and set shebang to null in it. I'm thus in favor of removing this code, as it seems very unlikely a user run in this case without knowing it.